Documentation/Contributing/Roadmap.md

```diff
@@ -4,14 +4,17 @@ This document outlines the roadmap of the Mixed Reality Toolkit.
 
 ## Current release
 
-[Microsoft Mixed Reality Toolkit v2.1.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/v2.1.0)
+[Microsoft Mixed Reality Toolkit v2.3.0](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases/tag/v2.3.0)
 
 ## Upcoming releases
 
 | Product | Description | Timeline | Project board |
 | --- | --- | --- | --- |
-| [MRTK V2.3](#230) | Next iteration of MRTK | January 2020 | https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10 |
-| [MRTK V2.4](#240) | Future iteration of MRTK | TBD | https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/11 |
+| [MRTK V2.4](#240) | Next iteration of MRTK | March 2020 | https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/11 |
+| [MRTK V2.5](#250) | Future iteration of MRTK | TBD 2020 | https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/11 |
```

Comment:
I don't think there's a lot of value to adding an arbitrary length list of releases without details (i.e. just having 2.4 is fine) along with maybe a mention that releases are roughly bi-monthly

Reply:
fair enough